test/claims-status/components/appeals-v2/Issues.unit.spec.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import Issues from '../../../../src/js/claims-status/components/appeals-v2/Issues';
+import { categorizeIssues } from '../../../../src/js/claims-status/utils/appeals-v2-helpers';
+import { mockData } from '../../../../src/js/claims-status/utils/helpers';
+
+describe('<Issues/>', () => {
+  const emptyIssues = { issues: categorizeIssues(mockData.data[0].attributes.issues) };
+  const oneIssues = { issues: categorizeIssues(mockData.data[1].attributes.issues) };
+  const manyIssues = { issues: categorizeIssues(mockData.data[2].attributes.issues) };
+
+  it('should render', () => {
+    const wrapper = shallow(<Issues {...emptyIssues}/>);
+    expect(wrapper.type()).to.equal('div');
+  });
+
+  it('should render no panels when no issues passed in', () => {
+    // Note: this probably isn't possible in real-world usage
+    const wrapper = shallow(<Issues {...emptyIssues}/>);
+    expect(wrapper.find('CollapsiblePanel').length).to.equal(0);
+  });
+
+  it('should render one panel when only an open *OR* a closed issue is passed in', () => {
```

Comment:
Should we test both panels separately rather than testing one and assuming the other works too?